app/src/test/kotlin/fr/free/nrw/commons/delete/ReasonBuilderTest.kt

```diff
@@ -56,7 +56,7 @@ class ReasonBuilderTest {
                 .thenReturn(Single.just(mock(FeedbackResponse::class.java)))
 
         val media = mock(Media::class.java)
-        `when`(media!!.dateUploaded).thenReturn(Date())
+        media?.dateUploaded=Date()
```

Comment:
mocks should not have their fields assigned. Also I don't believe `val media` could be nullable

Reply:
Yes, but, when with getter won't work for kotlin public val's and as the val needs to be public  for Room to generate the necessary classes, I was left with no other option than to manually assign values

Reply:
Then just create a Media object. Also this test passes if you pass in a null for the media object, that is preferrable

Reply:
I am not sure If passing a null object would be wise, creating the minimal media object for the same.